tests/sphinx/test_util.py

```diff
@@ -12,6 +12,7 @@
 
 
 class TestSphinxUtil(unittest.TestCase):
+    @unittest.skipIf('linux' not in sys.platform, "Running of the addon is only supported on linux")
```

Comment:
Actually, that the addon itself does not _run_ on other OS than Linux is fine for me. However, which version of a post-processing script is available should be also accessible from windows or mac. This would be in line with our other resources where we allow to build up a VASP job under windows and sent it to a remote machine (running on Linux) to be executed. Therefore, I would like this to be running OS independent. Can the script be converted to a python one?

Reply:
This is no problem at all. If anything else fails, I can work with wrappers to keep my favorite language on the local install.

But I have little experience with python under other OSs. What would the best way to run a python script in a portable way and capture the (json) output? Is for instance subprocess("script.py", shell=True) portable?

Reply:
Hmm, looking at the `ScriptJob` which runs python code we use `f'python {absolute_path_to_script}'` in the subprocess call:
https://github.com/pyiron/pyiron_base/blob/7ffb98d87f610967cd3c141d787bba7f83c687db/pyiron_base/jobs/script.py#L372-L400

And that is used in all OSs.